src/components/UniversalTable/UniversalTable.tsx

```diff
@@ -0,0 +1,20 @@
+import {  ReactNode } from 'react';
+import classes from './UniversalTable.module.scss';
+import { UniversalTableProps } from './types';
+
+const UniversalTable = <T,>({data, headers}:UniversalTableProps<T>):ReactNode => {
+  return (
+      <div className={classes.TableContainer}>
+        <ul className={classes.TableHeader}>
+          {headers.map((header)=> <li key={header.title} className={classes.TableCell}>{header.title}</li>)}
+        </ul>
+        {data.map((row, rowIndex) => (
+          <ul key={rowIndex} className={classes.TableRow}>
+            {headers.map((header, cellIndex)=> <li key={cellIndex} className ={classes.TableCell}>{row[header.property]}</li>)}
```

Comment:
I'd appreciate if you'd create a checking that every data item has an `id` props and throw an error if at least one data item doesn't have `id`. because using `cellIndex` as a key leads to possible bad side effects

Reply:
During our last call, you helped me create a generic for this component.

Question about ID we`ve have spoken about with Roman. But fased with some problems with generic. And i need some your help now

Reply:
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/0796a433-ae66-469f-9469-1a2d5cea3af0)
done (apply example of table)